{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "67baceed_c47eb85a",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 4
      },
      "lineNbr": 0,
      "author": {
        "id": 1179631
      },
      "writtenOn": "2021-11-15T06:12:21Z",
      "side": 1,
      "message": "Did you work out why this started failing? If so, could you put an explanation into the commit message and bug?\n\nIf not, IMO we shouldn\u0027t just change the test so it passes without an investigation of what changed to make it start failing.",
      "revId": "d2538efc492216f8374731e57cc5ead7fc255ae1",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}